README.md

```diff
@@ -1 +1,31 @@
-Repository to store performance and scalability-related testing scripts
+# FreeIPA Performance Testing
+
+Collection of performance and scalability-related testing scripts.
+
+## Dependencies
+
+This tool uses Vagrant and Libvirt to deploy VMs for testing.
+The controller must have the following software installed:
+
+* `vagrant`
+* `ansible`
+* `libvirt`
+* `vagrant-libvirt` (vagrant plugin)
+* `git`
+
+## Usage
+
+`./ipaperftest.py [OPTIONS]`
+
+* `test`: Test to perform. Current options are:
+  * `EnrollmentTest`: Try to enroll n clients simultaneously. This is the default.
+  * `APITest`: Try to run IPA command n times simultaneously.
+* `amount`: Size of the test. For example, in the case of `EnrollmentTest`, it will create this amount of clients. 
+* `command`: Command to run. Only relevant when using `APITest`. Use `{id}` for getting an unique ID into the command. Example: `ipa user-add user{id}`.
+* `client-image`: Vagrant image to use for the clients. Default: `antorres/fedora-34-ipa-client`. If you use a different image, make sure it has all the needed packages installed.
```

Comment:
Should we define the minimum required packages? I guess it may depend on the test, but I suppose it would be `[free]ipa-client `and `at`.

Reply:
Thanks, added. Also added `rsync` to the list of packages needed in the controller (it is used to collect logs).